lib/team.rb

```diff
@@ -1,20 +1,18 @@
 class Team
   attr_reader :team_id,
-              :franchise_id,
-              :team_name,
-              :abbreviation,
-              :stadium,
-              :link
+              :team_name
```

Comment:
YAGNI BABY - at least for now, we'll see what stats we need as we move on, might have to add more in later.

Reply:
I need to do this in the game class too!